### DISCUSSION TITLE
How to export and import collections with data?

### DISCUSSION THREAD

**Discussion by edikurniawan-dev** [pocketbase/pocketbase#2836] [Q&A] [answered]
I'm success to export and import collection but without data, how to export and import collections with data?

**Comment by ganigeorgiev** [comment, marked as answer]
There are no web APIs for collections data import/export at the moment (_eventually will be handled as part of #48, but before that we'll need to implement web bulk/transactions operations_).

If you want to export your collections with data, you can copy your `pb_data` OR create manual [migrations](https://pocketbase.io/docs/migrations/) (_if the collections you want to import have files then it is a little more complicated since it will depend from where you want to load the files; documentation for the js migrations will be available with the next release_).
